packages/template-block/src/TemplateBlock.tsx

```diff
@@ -38,6 +38,15 @@ export const TemplateBlock = ({ appBridge }: BlockProps): ReactElement => {
 
     const handleNewPublication = async () => {
         if (selectedTemplate !== null) {
+            const { templateEditing } = blockSettings;
+            if (templateEditing === TemplateEditing.Simple) {
+                const uri = `/publishing/template/${selectedTemplate.id}?referer=${window.location.href.replaceAll(
```

Comment:
I would prefer `encodeURI()` here

```suggestion
                const uri = `/publishing/template/${selectedTemplate.id}?referer=${encodeURI(window.location.href)}`
```